### PR TITLE
Update ENV config var

### PR DESCRIPTION
**Pull request by etruong42**
Read from explicitly PLAINTEXT connection string
Change log level back to INFO

### CODE REVIEW

**Comment by gurpreet-shah** [comment]
lgtm

File: scripts/heroku-run.bash [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # and
 # kafka://10.1.16.157:6667,kafka://10.1.0.165:6667,kafka://10.1.50.155:6667
 ZOOKEEPER="${HEROKU_KAFKA_ZOOKEEPER_URL//zookeeper:\/\/}"
-KAFKA="${HEROKU_KAFKA_URL//kafka:\/\//}"
+KAFKA="${HEROKU_KAFKA_PLAINTEXT_URL//kafka:\/\//}"
```
Comment (mkostick) [inline scripts/heroku-run.bash:18]: Does the plaintext URL also use `kafka://`?

**Comment by mkostick** [comment]
G2G as long as my other comment is OK.

File: scripts/heroku-run.bash (hunk 1 above)
Comment (etruong42) [reply scripts/heroku-run.bash:18]: Yep. And fyi, the SSL URL users `kafka+ssl://` scheme

**Pull request merged**